src/Framework/OperatingSystem.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if !NET5_0_OR_GREATER
+
+using System;
+
+namespace Microsoft.Build.Framework
+{
+    /// <summary>
+    /// System.OperatingSystem static methods were added in .Net 5.0.
+    /// This class create stand-in methods for net472 builds.
+    /// Assumes only Windows is supported.
+    /// </summary>
+    public static class OperatingSystem
```

Comment:
can this be internal instead of public? The API analyzer is complaining about this new public type missing from the reference assembly.

Reply:
yes - fixed.